art/attacks/inference/attribute_inference/true_label_baseline.py

```diff
@@ -59,6 +59,7 @@ def __init__(
         attack_model_type: str = "nn",
         attack_model: Optional["CLASSIFIER_TYPE"] = None,
         attack_feature: Union[int, slice] = 0,
+        is_regression: Optional[bool] = False,
```

Comment:
Would it be possible to derive this information from the type of `attack_model`?

Reply:
Does the type of `attack_model` have to be updated to include `REGRESSOR_TYPE`?